CHANGELOG.md

```diff
@@ -8,11 +8,21 @@
     - Added ability to choose deployment region (US, EU).
     - Added ability to force-upload an assembly.
     - Automatically open the SpatialOS Console page for a launched deployment.
+- New way to control tile distribution and types in the world via TileTypeVolume component :-
```

Comment:
nit: extra `-` at end of line after colon

Reply:
That wasn't a mistake, but you calling it out led to me learning something interesting about the use of a colon followed by a dash: https://english.stackexchange.com/questions/31060/is-it-proper-to-use-a-colon-followed-immediately-by-a-hyphen

In light of this new information, I shall amend the error of my ways and remove it.